ee/clickhouse/models/person.py

```diff
@@ -100,7 +100,7 @@ def create_person(
         "team_id": team_id,
         "properties": json.dumps(properties),
         "is_identified": int(is_identified),
-        "created_at": timestamp.isoformat(),
+        "created_at": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
```

Comment:
When this timestamp was in ISO string format it wasn't parsed properly from kafka topic -> CH kafka table

Reply:
(because of timezone inclusion)